documentation/APIbacklog.md

```diff
@@ -7,8 +7,8 @@
   | **API [family] name**  |  **API [family] owner** |  **API [family] proposal registration date**  | **Link to the filled-in application template**   |  **Supporters in API Backlog Working Group**  |  **API backlog - Status** |  **TSC - Status** | **Sub Project**  |
  | --- |  --- | --- | --- | --- | --- | --- | --- |
  | Quality on Demand | Deutsche Telekom | N/A | N/A | Deutsche Telekom, Orange, Telefónica, Vodafone|  --- |Approved (N/A) | [QualityOnDemand](https://github.com/camaraproject/QualityOnDemand) |%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Simple Edge Discovery --->
-| Simple Edge Discovery | Vodafone | N/A | N/A  | 5GFF, Vodafone | --- | Approved (N/A)  | [EdgeCloud/Discovery](https://github.com/camaraproject/EdgeCloud/tree/main/code/API_definitions/Discovery) |  <!---MEC Management --->
-| MEC Management | Telefónica | N/A | N/A  | GSMA-OGW, Telefónica | --- | Approved (N/A) |[EdgeCloud/Discovery](https://github.com/camaraproject/EdgeCloud/tree/main/code/API_definitions/Discovery)  | %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Device Location-->%%%%%%%%%%%%%%%%%%%%%
+| Simple Edge Discovery | Vodafone | N/A | N/A  | 5GFF, Vodafone | --- | Approved (N/A)  | [SimpleEdgeDiscovery](https://github.com/camaraproject/SimpleEdgeDiscovery) |  <!---MEC Management --->
+| MEC Management | Telefónica | N/A | N/A  | GSMA-OGW, Telefónica | --- | Approved (N/A) |[SimpleEdgeDiscovery](https://github.com/camaraproject/SimpleEdgeDiscovery)  | %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Device Location-->%%%%%%%%%%%%%%%%%%%%%

```

Comment:
My understanding is that only the SimpleEdgeDiscovery API will go into the new repository. Otherwise there will be again the challenge with multiple APIs with different lifecycle. I propose to keep line 11 unchanged.